### ISSUE TITLE
Parsing correctly?

### ISSUE THREAD

**Issue by robert-blair** [rollup/rollup-plugin-commonjs#172]
Im trying to run roll-up in tandem with the angular aot compiler, the aot works fine but when I try to run roll-up it says that there's an unexpected error in an rxjs file, but I don't think it's there??

Could there be something wrong with the parser?

![screenshot](https://cloud.githubusercontent.com/assets/17201531/23665620/d45eb64a-030d-11e7-93ab-bec6fcbac83a.jpg)

```
import rollup      from 'rollup'
import nodeResolve from 'rollup-plugin-node-resolve'
import commonjs    from 'rollup-plugin-commonjs';
import uglify      from 'rollup-plugin-uglify'


export default {
  entry: 'src/main-aot.js',
  dest: 'aot/dist/build.js',
  sourceMap: true,
  sourceMapFile: 'aot/dist/build.js.map',
  format: 'iife',
  onwarn: function(warning) {
    
    if ( warning.code === 'THIS_IS_UNDEFINED' ) { return; }
    if ( warning.indexOf("The 'this' keyword is equivalent to 'undefined'") > -1 ) { return; }


    console.warn( warning.message );
  },
  
  plugins: [
    nodeResolve({jsnext: true, module: true}),
    commonjs({include: ['node_modules/rxjs/**']}),
    uglify()
  ]
}
```

**Comment by Rich-Harris** [comment]
I think this might have been fixed by #170 — could you try reinstalling this plugin and seeing if it's still an issue? Thanks

**Comment by robert-blair** [comment]
Yup that fixed it.

Thank you.
